Jenkinsfile

```diff
@@ -63,13 +59,6 @@ builders = pipeline_builder.createBuilders { container ->
             }
 
             def configure_epics = ""
```

Comment:
I assume this line and its use below can also be removed?

Reply:
True, I'll delete them.